playbooks/templates/suricata_overrides.yaml.j2

```diff
@@ -30,7 +30,8 @@ outputs:
   - eve-log:
       enabled: yes
       filetype: regular
-      filename: eve.json
+      filename: eve-%Y-%m-%d-%H.json
```

Comment:
You're reading in `eve.json` in the filebeat config, but this creates a date modified filename for suricata. Currently we're using the generic log rotation approach [recommended](http://suricata.readthedocs.io/en/latest/configuration/log-rotation.html) by the Suricata team in the [logrotate-suricata.conf](https://github.com/rocknsm/rock/blob/64f2fccc176b41b6ffefb49c48e6564916bbcaa9/playbooks/templates/logrotate-suricata.conf) template.

I'm not tied one way or the other, I just want to ensure its as consistent as possible and everything lined up across the different configs.